integrationTests/p2p/antiflood/messageProcessor.go

```diff
@@ -0,0 +1,79 @@
+package antiflood
+
+import (
+	"fmt"
+	"sync"
+	"sync/atomic"
+
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+type messageProcessor struct {
+	numMessagesProcessed  uint32
+	sizeMessagesProcessed uint64
+
+	numMessagesReceived  uint32
+	sizeMessagesReceived uint64
+
+	mutMessages    sync.Mutex
+	messages       map[p2p.PeerID][]p2p.MessageP2P
+	FloodPreventer process.FloodPreventer
```

Comment:
is this a shared resource ? you can have a getter for it.

Reply:
Not shared. Made it unexported as it can be accessed in the same package. It can be overwritten in tests with nil as to disable anti flooding capabilities (for flooder peers). Since we are in integration test I think it is overkill to create Set/Get.